CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [12.18.1]
+### Changed
+- Update Solana and Evm metrics emitted when bridging ([#33056](https://github.com/MetaMask/metamask-extension/issues/33056))
+
+### Fixed
+- Allow users to load ERC20 tokens automatically while Activity or NFT tab are open ([#33014](https://github.com/MetaMask/metamask-extension/issues/33014))
```

Comment:
Looks like the URL is wrong

```suggestion
- Allow users to load ERC20 tokens automatically while Activity or NFT tab are open ([#33014](https://github.com/MetaMask/metamask-extension/pull/33014))
```

The rest are wrong as well, they all say `issues`

Reply:
Good catch ! added 9bfac17f9e2e81fdcf5ab8b873374d5a334a08e5